extensions/3DTILES_metadata/examples/terrainpack/org-tiledata.gltf

```diff
@@ -0,0 +1,1243 @@
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 560,
+      "byteLength": 160,
+      "byteStride": 8,
+      "target": 34962
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 720,
+      "byteLength": 160,
+      "byteStride": 8,
+      "target": 34962
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 880,
+      "byteLength": 160,
+      "byteStride": 8,
+      "target": 34962
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 1040,
+      "byteLength": 216,
+      "target": 34963
+    }
+  ],
+  "accessors": [
+    {
+      "bufferView": 0,
+      "byteOffset": 0,
+      "type": "VEC3",
+      "componentType": 5126,
+      "count": 20,
+      "min": [
+        -11321.248046875,
+        -7760.14453125,
+        13.74415111541748
+      ],
+      "max": [
+        -8029.4501953125,
+        -3640.53955078125,
+        61.3035888671875
+      ]
+    },
+    {
+      "bufferView": 1,
+      "byteOffset": 0,
+      "type": "VEC2",
+      "componentType": 5126,
+      "count": 20,
+      "min": [
+        0.040462516248226166,
+        0.03869234770536423
+      ],
+      "max": [
+        0.04424590989947319,
+        0.0432811863720417
+      ]
+    },
+    {
+      "bufferView": 2,
+      "byteOffset": 0,
+      "type": "VEC2",
+      "componentType": 5126,
+      "count": 20,
+      "min": [
+        0.2985794246196747,
+        0.570982038974762
+      ],
+      "max": [
+        0.357493132352829,
+        0.6541822552680969
+      ]
+    },
+    {
+      "bufferView": 3,
+      "byteOffset": 0,
+      "type": "VEC2",
+      "componentType": 5126,
+      "count": 20,
+      "min": [
+        0.29944366216659546,
+        0.5706337690353394
+      ],
+      "max": [
+        0.3581045866012573,
+        0.6534256935119629
+      ]
+    },
+    {
+      "bufferView": 4,
+      "byteOffset": 0,
+      "type": "VEC2",
+      "componentType": 5126,
+      "count": 20,
+      "min": [
+        0.2985794246196747,
+        0.570982038974762
+      ],
+      "max": [
+        0.357493132352829,
+        0.6541822552680969
+      ]
+    },
+    {
+      "bufferView": 5,
+      "byteOffset": 0,
+      "type": "VEC2",
+      "componentType": 5126,
+      "count": 20,
+      "min": [
+        0.2987642288208008,
+        0.5709075331687927
+      ],
+      "max": [
+        0.3576238751411438,
+        0.654020369052887
+      ]
+    },
+    {
+      "bufferView": 6,
+      "byteOffset": 0,
+      "type": "SCALAR",
+      "componentType": 5125,
+      "count": 54
+    }
+  ],
+  "meshes": [
+    {
+      "primitives": [
+        {
+          "mode": 4,
+          "indices": 6,
+          "attributes": {
+            "POSITION": 0,
+            "TEXCOORD_0": 1,
+            "TEXCOORD_1": 2,
+            "TEXCOORD_2": 3,
+            "TEXCOORD_3": 4,
+            "TEXCOORD_4": 5
+          },
+          "material": 0,
+          "extensions": {
+            "EXT_3dtiles_feature_metadata": {
+              "featureLayers": [
```

Comment:
Just to double check: it's okay to use featureTable/featureProperties because we're using the old `EXT_3dtiles_feature_metadata` spec, correct?